app/helpers/admin/injection_helper.rb

```diff
@@ -16,7 +16,7 @@ def admin_inject_enterprise_relationships
     end
 
     def admin_inject_enterprise_roles
-      admin_inject_json_ams_array "ofn.admin", "enterpriseRoles", @enterprise_roles, Api::Admin::EnterpriseRoleSerializer
+      admin_inject_json_ams_array "admin.enterpriseRoles", "enterpriseRoles", @enterprise_roles, Api::Admin::EnterpriseRoleSerializer
```

Comment:
IMHO creating a whole new AngularJS module and dependency (`admin.enterpriseRoles`) for the purposes of providing one value (`enterpriseRoles`) is overkill. You can just provide this value directly to the `admin.enterprises` module, and lose `admin.enterpriseRoles` module.

Reply:
Actually, scrap that. I see what you are doing now (sorry, took me a bit to catch up). See my comment [above](https://github.com/openfoodfoundation/openfoodnetwork/pull/1720/files#r148704686) for regarding the use of the `admin.enterpriseRoles` module. If that can happen, then I like the approach you've taken here.

Reply:
I think I initially tried to include it in `admin.enterprises` but there were complicated conflicts with using it in different contexts or alongside other modules. Or something like that. Separating it out into an independent module fixed these issues but I can't remember all the details.

Reply:
Great. I think there is value is keeping this code in a separate module. I would like to do a little refactor/cleanup at some point as per comment referenced above but I don't think it's important for now.